File: frameworks/Elixir/plug/lib/framework_benchmarks/handlers/plain-text.ex
defmodule FrameworkBenchmarks.Handlers.PlainText do
  @moduledoc """
  This is the handle for the /plaintext route
  """
  def handle(conn) do
    conn
    |> Plug.Conn.put_resp_content_type("text/plain")
    |> Plug.Conn.send_resp(200, "Hello, World!")
  end
end

File: frameworks/CSharp/effinitive/Benchmarks/Benchmarks/Plaintext.cs
using System.Threading;
using System.Threading.Tasks;

using EffinitiveFramework.Core;

namespace Benchmarks.Benchmarks;

public class Plaintext : NoRequestEndpointBase<string>
{
    protected override string Method => "GET";

    protected override string Route => "/plaintext";

    protected override string ContentType => "text/plain";

    public override ValueTask<string> HandleAsync(CancellationToken cancellationToken = default)
    {
        return ValueTask.FromResult("Hello, World!");
    }

}
